paasta_tools/contrib/kill_bad_containers.py

```diff
@@ -62,6 +62,9 @@ def kill_containers_with_duplicate_iptables_rules(docker_client):
                 print(container2)
                 print("Not sure which to kill, picking the first one")
                 docker_client.kill(container1["Id"])
+            print("Deleting the iptables rules")
+            chain.delete_rule(iptables_rule)
+            chain.delete_rule(raw_rules[target])
```

Comment:
If it's two different containers, and we only kill one of them, don't we need to keep one of the rules?

Reply:
Hmm right. Crap.